### PR TITLE
change success message in BlogController

### PR DESCRIPTION
**Pull request by abiodunmale**
change the message on both function delete and update to include successfully

### CODE REVIEW

File: app/Http/Controllers/BlogController.php [hunk 1]
```diff
@@ -113,7 +113,7 @@ public function update(Request $request, $slug)
             ]);
 
         return redirect(RouteServiceProvider::HOME)
-            ->with('message', 'Your post has been updated!');
+            ->with('message', 'Your post has been updated successfully!');
```
Comment (folorunsofavour) [inline app/Http/Controllers/BlogController.php]: Make the "successfully" "Successfully"

File: app/Http/Controllers/BlogController.php [hunk 2]
```diff
@@ -123,6 +123,6 @@ public function delete($slug)
         $blog->delete();
 
         return redirect('/blog')
-            ->with('message', 'Your post has been deleted!');
+            ->with('message', 'Your post has been deleted successfully!');
```
Comment (folorunsofavour) [inline app/Http/Controllers/BlogController.php]: Also make the "successfully" "Successfully"

**Review by folorunsofavour — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/BlogController.php (hunk 1 above)
Comment (abiodunmale) [reply app/Http/Controllers/BlogController.php]: you should have changed the pull title to something like "change success message in BlogController ....."
be direct cos that the essence of review 
what des "Make the "successfully" "Successfully"" means. even me I don't understand

Comment (folorunsofavour) [reply app/Http/Controllers/BlogController.php]:
```suggestion
            ->with('message', 'Your post has been updated Successfully!');
```

File: app/Http/Controllers/BlogController.php (hunk 2 above)
Comment (folorunsofavour) [reply app/Http/Controllers/BlogController.php]: Change the "successfully" in the success message in BlogController to "Successfully"

**Review by folorunsofavour — CHANGES_REQUESTED** [review]

**Comment by folorunsofavour** [comment]
can you give a detailed description of what you did here?

**Review by folorunsofavour — APPROVED** [review]
Great Job

**Pull request merged**